utils/install-basic-software.ps1

```diff
@@ -32,16 +32,25 @@ $packages = @(
 echo "Setting up Chocolatey software package manager"
 Get-PackageProvider -Name chocolatey -Force
 
+echo "Setting up Full Chocolatey Install"
+Install-Package -Name Chocolatey -Force -ProviderName chocolatey
+$chocopath = (Get-Package chocolatey | ?{$_.Name -eq "chocolatey"} | Select @{N="Source";E={((($a=($_.Source -split "\\"))[0..($a.length - 2)]) -join "\"),"Tools\chocolateyInstall" -join "\"}} | Select -ExpandProperty Source)
+& $chocopath "upgrade all -y"
+choco uninstall mm-choco.extension --all-versions
```

Comment:
I figured out why I had that in there. A while back the Notepad++ package had a dependency on that legacy extension which has since been fixed in the current package. You can safely remove that line.